tests.js

```diff
@@ -0,0 +1,53 @@
+const capitalize = (string) => string[0].toUpperCase() + string.slice(1);
+
+const reverse = (string) => string.split('').reverse().join('');
+
+const Calculator = (() => {
+  const add = (a, b) => (a + b);
+
+  const subtract = (a, b) => a - b;
+
+  const divide = (a, b) => a / b;
+
+  const multiply = (a, b) => a * b;
+
+  return {
+    add, subtract, divide, multiply,
+  };
+})();
+
+const caesar = (str, k) => {
+  const check1 = (newcode) => String.fromCharCode(((newcode - 122) % 26) + 96);
+
+  const check2 = (newcode) => String.fromCharCode(newcode);
+
+  const check3 = (newcode) => String.fromCharCode(((newcode - 90) % 26) + 64);
```

Comment:
[Optional]

We could give `check1`, `check2` and `check3` a more descriptive name to make the function more readable.